Classes/Service/DyncssService.php

```diff
@@ -103,4 +103,18 @@ public static function getOverrides()
 
         return $overrides;
     }
+
+    /**
+     * Check CLI mode depending on TYPO3 version
+     *
+     * @return int
+     */
+    protected static function isCliMode()
+    {
+        if (version_compare(TYPO3_version, '8.0', '<')) {
+            return TYPO3_cliMode;
```

Comment:
I'd assume this will still trigger https://github.com/kaystrobach/TYPO3.dyncss/issues/54